Sources/Rownd/Models/Context/Passkeys.swift

```diff
@@ -27,7 +27,7 @@ public struct PasskeyState: Hashable {
     public var isInitialized: Bool = false
     public var isErrored: Bool = false
     public var errorMessage: String?
-    public var registration: [PasskeyRegistration]? = []
+    public var registration: [PasskeyRegistration] = []
```

Comment:
Since it's now non-optional, let's double-check that this doesn't cause any state parsing issues on network loads or when moving from a previous SDK version to a this SDK version.

Reply:
Oh good catch. I added the optional back.